common/test/acceptance/pages/lms/teams.py

```diff
@@ -315,6 +322,13 @@ def team_user_membership_text(self):
         query = self.q(css='.page-content-secondary > .team-user-membership-status')
         return query.text[0] if query.present else ''
 
+    def wait_for_leave_team_button_presence(self):
+        """ Helper method to wait for Leave Team button to appear """
+        self.wait_for(
```

Comment:
Is there a reason this isn't `self.wait_for_element_presence('.leave-team-link', '... message ...')`?